umpf

```diff
@@ -1290,7 +1291,8 @@ create_tag() {
 	fi
 
 	tree="$(${GIT} log -1 --format="%T")"
-	for topic in $(<"${STATE}/topics"); do
+	mapfile -t topics < <(cat ${STATE}/topics 2>/dev/null)
```

Comment:
```suggestion
	mapfile -t topics 2>/dev/null < ${STATE}/topics
```
Should work as well, I think. No need to use `cat`.

Reply:
That would just silence the error. `run_tag()` has `set -e`, so this still aborts.

Reply:
In that case, just make the whole thing conditional. The loop is trivial and an extra indention level won't hurt here.